### ISSUE TITLE
Account creation: focusing the interactive elements

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-in-action/platform#936]
Safari/ Keyboard navigation > Account creation flow: 
Once you go to the "Your detail" page, the "Full name" filed automatically gets the focus. This is a great experience, however, that doesn't happen in the other pages of account creation. On other pages you have to tab through many other elements to get to the filed or element that you need to make a selection.

**Comment by greatislander** [comment]
I'm not sure that focusing the first input on the page is the best practice. If it is, this should be done on every form.
